src/components/CreateEnvironment/PackageSelect/Listbox/index.tsx

```diff
@@ -94,7 +94,7 @@ function renderRow(props: ListChildComponentProps) {
   );
 }
 
-function useResetCache(data: any) {
+function useResetCache<T>(data: T) {
```

Comment:
No need for generics here; I'm assuming that useEffect has an actual type that is used for the effect dependency, you should probably use that. If it doesn't, just use `unknown`.

Reply:
`unknown` works fine, thanks! (`useEffect` takes a `React.DependencyList`, which is just `unknown[]`)

Reply:
Of course is it.